tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/OutputMode.scala

```diff
@@ -36,33 +36,38 @@ object OutputMode {
     val esLevel: ESLevel = ESLevel.ES5
   }
 
-  /** Modern output mode compliant with ECMAScript 5.1 in a function scope.
-   *  This is the default output mode used by fastOpt and fullOpt.
-   *  The output must be enclosed in an anonymous function isolating the code
-   *  in a dedicated scope.
+  /** Output mode compliant with ECMAScript 5.1 (deprecated alias).
+   *
+   *  This value is not annotated with `@deprecated` for technical reasons, but
```

Comment:
The reason is that one cannot deprecate a `class` or `object` for the purpose of *constructing* it (which is the intended usage for users) without deprecating pattern matching on it or more generally *deconstructing* it, which we internaly need to keep supporting the feature. At least not in a binary compatible way.

For this class I'm not worried that users actually ever wrote it anywhere, because the only documented/advertised way to create it is via the `RuntimeDOM` DSL. So getting the word out is not an issue.